app/controllers/boards_controller.rb

```diff
@@ -1,4 +1,6 @@
 class BoardsController < ApplicationController
+  before_filter :authorize
```

Comment:
Ok, zmieniłem nazwę metody na `authenticate`. No tak bo authorization to już bardziej pundit